app/models/uk_market_conformity_assessment_body.rb

```diff
@@ -0,0 +1,46 @@
+class UkMarketConformityAssessmentBody < Document
+  validates :uk_market_conformity_assessment_body_name, presence: true
+  validates :updated_at, presence: true, date: true
+  validates :uk_market_conformity_assessment_body_number, presence: true
+  validates :uk_market_conformity_assessment_body_type, presence: true
+  validates :uk_market_conformity_assessment_body_registered_office_location, presence: true
+  validates :uk_market_conformity_assessment_body_testing_locations, presence: true
+  validates :uk_market_conformity_assessment_body_email, presence: true
+  validates :uk_market_conformity_assessment_body_phone, presence: true
+  validates :uk_market_conformity_assessment_body_legislative_area, presence: true
+
+  FORMAT_SPECIFIC_FIELDS = %i(
+    uk_market_conformity_assessment_body_name
+    updated_at
```

Comment:
I'm not sure that this should be here, since updated at shows up in other specialist documents.